src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml

```diff
@@ -187,10 +49,10 @@ InspectorSectionView {
                 width: parent.width
 
                 navigation.panel: root.navigationPanel
-                navigation.name: "More page settings"
+                navigation.name: "Page settings"
                 navigation.row: root.navigationRow(6)
 
-                text: qsTrc("inspector", "More page settings")
+                text: qsTrc("inspector", "Page settings")
```

Comment:
Note that there is already a translation for "Page settings", but not "More page settings"

Reply:
@Tantacrul what do you think of this change?

Reply:
Does this relate to the button that triggers the more settings dialog? If so, I don't exactly know why we'd add the word 'more'. @wizofaus Pretty much all page settings are in that dialog, no?

Perhaps I misunderstanding the string change

Reply:
You agreed none of the page-specific settings that were there actually made much sense to be directly on the inspector, so I removed them all, hence that button is now for *all* page settings (not just ones in addition to whatever was there before).
Also note there's no translation for "More page settings", but there is for "Page settings".

Reply:
I did agree to all those things. I think I misunderstood the string change. I thought it was the other way around. Apologies. Rushing